sdk/cognitivelanguage/Azure.AI.Language.sln

```diff
@@ -22,6 +19,10 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.AI.Language.Conversat
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.AI.Language.Conversations.Tests", "Azure.AI.Language.Conversations\tests\Azure.AI.Language.Conversations.Tests.csproj", "{E30B7233-76F8-4966-9CDC-86D9C5B37462}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.Core", "..\core\Azure.Core\src\Azure.Core.csproj", "{7DBAEA65-0607-404C-A326-8089FC2887CF}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.CognitiveServices.Knowledge.QnAMaker", "..\cognitiveservices\Knowledge.QnAMaker\src\Microsoft.Azure.CognitiveServices.Knowledge.QnAMaker.csproj", "{46AFA6FD-6AC0-4B23-8620-7340120BAA18}"
+EndProject
```

Comment:
Why was this added? Unless it's used for migration samples (and I don't see any), we typically don't want track 1 libraries in our solutions (waste of time to build).

Also, don't add Azure.Core to the solution unless required. I know there are some cases where it is, and if it was causing problems not being in the solution then, by all means, add it. :smile:

Reply:
I was getting errors because of not adding it, will remove it once we make sure that there are no changes.

Reply:
If Azure.Core was necessary, go ahead and leave it in. Or were you talking about the track 1 library?

Reply:
I was experimenting for both of them because I had errors. But now I tried again and only Azure.Core is necessary to eliminate the errors.